pkg/utils/urls/replace.go

```diff
@@ -21,5 +21,25 @@ func ReplaceHost(orgURL, host string) string {
 		u.Scheme = ""
 	}
 	u.Host = host
-	return strings.TrimPrefix(u.String(), "//")
+	return strings.ReplaceAll(strings.TrimPrefix(u.String(), "//"), "%2F", "/")
```

Comment:
this seems hacky. We should change the implementation of this method so `%2F` is not added